Sources/AccountSDKIOSWeb/Simplified Login/View/Sub Views/ContinueButton.swift

```diff
@@ -33,16 +33,53 @@ final class ContinueButton: UIButton {
         return view
     }()
 
+    private lazy var nameTitleLabel: UILabel = {
+        let view = UILabel()
+        view.text = title
+        view.isAccessibilityElement = false
+        view.font = UIFont.preferredFont(forTextStyle: .callout).bold()
+        view.textAlignment = .left
+        view.numberOfLines = 1
+        view.lineBreakMode = .byTruncatingTail
+        view.textColor = .white
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        return view
+    }()
+    
+    private lazy var combinedTitleView: UIView = {
+        return UIView()
+    }()
+
+    private lazy var emailLabel: UILabel = {
+        let view = UILabel()
+        view.text = viewModel.email
+        view.isAccessibilityElement = false
+        view.font = UIFont.preferredFont(forTextStyle: .footnote)
+        view.textAlignment = .left
+        view.lineBreakMode = .byTruncatingTail
+        view.numberOfLines = 1
+        view.textColor = .white
+        view.translatesAutoresizingMaskIntoConstraints = false
+
+        return view
+    }()
+
     private lazy var internalConstraints: [NSLayoutConstraint] = {
         [initialsLabel.centerYAnchor.constraint(equalTo: avatarView.centerYAnchor),
          initialsLabel.centerXAnchor.constraint(equalTo: avatarView.centerXAnchor),
-         avatarView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 12),
+         avatarView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 8),
          avatarView.centerYAnchor.constraint(equalTo: centerYAnchor),
          avatarView.widthAnchor.constraint(equalToConstant: 40),
          avatarView.heightAnchor.constraint(equalToConstant: 40),
-         titleLabel!.leadingAnchor.constraint(equalTo: avatarView.trailingAnchor, constant: 10),
-         titleLabel!.topAnchor.constraint(greaterThanOrEqualTo: topAnchor, constant: 3),
-         titleLabel!.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -3)
+         nameTitleLabel.leadingAnchor.constraint(equalTo: avatarView.trailingAnchor, constant: 8),
```

Comment:
Suggestion:

I usually add a small DSL extension to projects for handling autolayout constraints. (Kinda like in [this blog post](https://www.swiftbysundell.com/articles/building-dsls-in-swift/). 

Not for this PR but maybe something we can look into as a refactoring thing?

Reply:
Yes, we can think about it but maybe when we decide which version of SL is our final version? :)